ajax/libs/artDialog/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "artDialog",
+  "filename": "dialog-min.js",
+  "version": "6.0.4",
+  "description": "经典的网页对话框组件，内外皆用心雕琢",
+  "homepage": "https://aui.github.io/artDialog/",
+  "keywords": [
+    "dialog",
+    "ui",
+    "jquery"
```

Comment:
Where did you get `"ui"`, and `"jquery"`? I don't think these are good keywords.